docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -1196,9 +1162,36 @@ The following table lists all the search category attributes that the server can
 | `MC_ABILITY_CLIENT_CUSTOM`       | Ability for custom event                 |
 | `MC_ABILITY_SEARCH`              | Ability for search                       |
 
-> **Note**
->
-> These attributes support Tizen 5.5 and Higher for Mobile and Wearable.
+> [!NOTE]
+> These attributes support Tizen 5.5 and Higher.
+
+## Media Controller Result Code Attributes
+
+The following table lists all the result code attributes that the client and server can receive:
+
+**Table: Media controller result code attributes**
+
+| Attribute                               | Description                                             |
+|-----------------------------------------|---------------------------------------------------------|
+| **Result code**                         |                                                         |
+| `MC_RESULT_CODE_SUCCESS`                | The command or the event has completed successfully |
+| `MC_RESULT_CODE_ALREADY_DONE`           | The command or the event had executed previously |
+| `MC_RESULT_CODE_ABORTED`                | The command or the event is aborted by some external event (for example aborted play command by incoming call) |
+| `MC_RESULT_CODE_DENIED`                 | The command or the event is denied due to application policy (for example cannot rewind in recording) |
```

Comment:
Would you please add commas after "for example" also?

for example,

Reply:
Commas added after "for example".
Thanks!